discovery-provider/src/tasks/index_aggregate_user.py

```diff
@@ -341,79 +335,67 @@
     """
 
 
-def update_aggregate_table(
-    db,
-    redis,
-    table_name,
-    most_recent_indexed_aggregate_block_key,
-    query,
-    timeout=DEFAULT_UPDATE_TIMEOUT,
-):
+def _update_aggregate_table(session):
```

Comment:
Maybe we should rename this to be _update_aggregate_table or _update_aggregate_user too

Reply:
Updated the branch with the new function name.